pom.xml

```diff
@@ -59,47 +59,54 @@
       <artifactId>gson</artifactId>
       <version>2.3.1</version>
     </dependency>
+    <dependency>
+      <groupId>me.moocar</groupId>
+      <artifactId>socket-encoder-appender</artifactId>
+      <version>0.1beta1</version>
+    </dependency>
```

Comment:
It's only got one class and no new external deps. I would just include it by default, as I think a lot of users will want to use the TCP functionality given the choice.

If not, may I suggest that you add the dependency information for it on the README for this module?

By the way, awesome job on this project.